src/scheduler/workers/unpublish/weekly/__tests__/findReminderToSend.test.ts

```diff
@@ -0,0 +1,72 @@
+import { findReminderToSend } from "../findReminderToSend";
+
+const weeklyReminders = {
+  1: "2023-02-08T00:00:00.000Z",
+  2: "2023-02-15T00:00:00.000Z",
+  3: "2023-02-22T00:00:00.000Z",
+  4: "2023-03-01T00:00:00.000Z",
+  5: "2023-03-08T00:00:00.000Z",
+};
```

Comment:
Why an object instead of an array? I mean, they're both the same thing in JS but it looks a bit odd.

Reply:
what @alisalaman-cyb said 🙂